src/include/distributed/commands/grant.h

```diff
@@ -0,0 +1,19 @@
+/*-------------------------------------------------------------------------
+ *
+ * grant.c
```

Comment:
grant.h

Reply:
is it better to group these tiny header files together ?  We literally have a single function declared in many of those.

Reply:
That is a good question. I don't know if there are any real downsides of having multiple small header files vs a combined one.

Benefit of having them separated is that all functions related to this concept should be implemented here. It might be that I couldn't find all the functions just yet for granting permission to a table as I have only focused on the utilities file.

For other concepts you need to implement multiple scenarios. Creation before table distribution, creation after table distribution and new shard creation. I expect such functions to reside all together in this file in the end. At what point would it warrant to split them in their own header file?